waiter/src/waiter/state/responder.clj

```diff
@@ -30,7 +30,8 @@
             [waiter.util.date-utils :as du]
             [waiter.util.utils :as utils]
             [waiter.work-stealing :as work-stealing])
-  (:import clojure.lang.PersistentQueue))
+  (:import clojure.lang.PersistentQueue
+           (org.joda.time DateTime)))
```

Comment:
Is this import unused? If so, please remove.

Reply:
Removed